requirements.txt

```diff
@@ -1,3 +1,4 @@
 torch==1.4.0
 scipy==1.4.1
+sklearn
```

Comment:
Please specify your version. Thus we can be sure that the lib will work after the latest version changes